module/Finna/src/Finna/View/Helper/Root/Feed.php

```diff
@@ -67,8 +67,10 @@ public function __construct(\Laminas\Config\Config $config)
      */
     public function __invoke($id)
     {
-        if (isset($this->config[$id]) && $this->config[$id]['active']) {
-            return $this->getView()->render('Helpers/feed.phtml', ['id' => $id]);
+        $feedConfig = $this->config[$id];
+        if (isset($feedConfig) && $feedConfig['active']) {
```

Comment:
nullcoalesce lyhentäis

Reply:
Joo, käytetään sitä ennemmin.

Reply:
Siis "??" mihin liittyen? Näin?:
`$feedConfig = $this->config[$id] ?? '';`
`if ($feedConfig && $feedConfig['active']) {`
`...`